src/__tests__/App.test.js

```diff
@@ -0,0 +1,15 @@
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import App from '../App';
+
+describe('App.js', () => {
+  it('should render correctly', () => {
+    const tree = renderer.create(
+      <Provider store={store}>
+        <App />
+      </Provider>,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
- [ ] This test case is failing, with the error pasted in the screenshot below
<img width="904" alt="Screenshot 2023-06-01 at 17 02 20" src="https://github.com/yusufobr/crypto-data/assets/56886064/c3e6bae2-0f90-466f-81f8-aa15abacc7fa">

Reply:
done